scripts/valhalla_build_config

```diff
@@ -33,6 +33,7 @@ config = {
         'transit_dir': '/data/valhalla/transit',
         'transit_feeds_dir': '/data/valhalla/transit_feeds',
         'transit_bounding_box': Optional(str),
+        'transit_pbf_limit': 20000,
```

Comment:
those are representing gtfs trips. let's see what real feeds do. thanks @kevinkreiser for suggesting that.